compiler/src/iree/compiler/API/Internal/CompilerDriver.cpp

```diff
@@ -169,6 +170,57 @@ class rt_aligned_string_ostream : public llvm::raw_ostream {
   }
 };
 
+llvm::ThreadPoolStrategy getGlobalThreadPoolStrategy() {
+  // We allow two environment variables to control the compiler thread pool.
+  //   IREE_COMPILER_THREAD_COUNT: If present (and not empty or 0), will
+  //     create a thread pool with the requested number of threads, even
+  //     if this is unwise or beyond the limit limit of hardware concurrency.
+  //   IREE_COMPILER_TASK_COUNT: Specifies a target maximum number of
+  //     concurrent tasks to support at any given time. The actual number
+  //     of threads will be limited to the hardware concurrency if in
+  //     excess. If zero, then the hardware concurrency is used.
+  // In the absence of any environment variables, the behavior is as if
+  // IREE_COMPILER_TASK_COUNT was specified with a hard-coded value that
+  // has been chosen by project developers. If a number greater than
+  // zero IREE_COMPILER_THREAD_COUNT will take precedence.
```

Comment:
Does this mean if both of flags are specified, IREE_COMPILER_THREAD_COUNT will be used? Can we make it a bit more clear?

Reply:
I just removed THREAD_COUNT. I had added it to test the effect of extreme oversubscription of the hardware concurrency, but as expected, that is just a bad idea.